cartoframes/examples.py

```diff
@@ -0,0 +1,17 @@
+"""example data factory"""
+from cartoframes import CartoContext
+
+EXAMPLE_BASE_URL = 'https://cartoframes.carto.com'
+EXAMPLE_API_KEY = 'default_public'
+# EXAMPLE_BASE_URL = 'http://cdb.localhost.lan:8888/'
+# EXAMPLE_API_KEY = 'default_public'
+
+
+class Examples(CartoContext):
```

Comment:
I'm in the class camp now because we can overwrite the CartoContext methods that we don't want to expose 👍 

Something like the following for `write`, `data*`, and `delete`:

```python
def data(self):
    raise RuntimeError('CartoContext.data disabled for the Examples CartoContext')
```